sdk/servicebus/service-bus/src/serviceBusMessageBatch.ts

```diff
@@ -69,7 +74,10 @@ export interface ServiceBusMessageBatch {
    * @param message - An individual service bus message.
```

Comment:
Does AmqpAnnotatedMessage count as a service bus message, or do we need more verbiage?

Reply:
The type info for the parameter should be enough right?

Reply:
We could skip the term "service bus" and just say "message". 

> An individual message

or

> The message to add to the batch.

Reply:
The message to add to the batch.